ibm/resource_ibm_is_vpc.go

```diff
@@ -89,6 +90,14 @@ func resourceIBMISVPC() *schema.Resource {
 				Description: "Default network ACL",
 			},
 
+			isVPCDefaultRoutingTable: {
+				Type:        schema.TypeString,
```

Comment:
isVPCDefaultRoutingTable here should be only computed  and we need to set in Read Method similar to datasource